src/main/java/app/knock/api/model/UpdateSchedulesRequest.java

```diff
@@ -56,6 +55,26 @@ public UpdateSchedulesRequestBuilder addActor(String actor) {
             return this;
         }
 
+        public UpdateSchedulesRequestBuilder addActor(Map<String, Object> actor) {
```

Comment:
I think this is something we missed, as `actors` can also be passed as a map full of custom properties.